src/components/SliderMenu.tsx

```diff
@@ -7,13 +7,15 @@ import styled, { useTheme } from "styled-components";
 
 interface SliderMenuProps {
   title: string;
+  hasNav?: boolean;
   isOpen: boolean;
   onClose?: () => void;
   children?: React.ReactNode;
 }
 
 export default function SliderMenu({
   title,
+  hasNav,
```

Comment:
In the Send view, the last option inside the menus was covered by the bottom navigation.